### PR TITLE
updated slepsc and petsc packages.

### PR DESCRIPTION
**Pull request by alarshi**
This PR adds the updated packages for slepsc and petsc.

### CODE REVIEW

**Review by tjhei — APPROVED** [review]

File: deal.II-toolchain/packages/petsc.package [hunk 1]
```diff
@@ -2,8 +2,11 @@
 ## PETSc                                                                      ##
 ################################################################################
 
-VERSION=3.19.0
-CHECKSUM=3d8820670f0d67aa04ce41b91f8eae62
+VERSION=3.19.3
```
Comment (tjhei) [inline deal.II-toolchain/packages/petsc.package:5]: Can you update to 3.19.4? I think it just got released.

File: deal.II-toolchain/packages/petsc.package [hunk 2]
```diff
@@ -2,8 +2,11 @@
...
+CHECKSUM=b493f0c19c067994ce7e9b5f4d13216c
+
+#VERSION=3.19.0
```
Comment (tjhei) [inline deal.II-toolchain/packages/petsc.package:8]: You can delete this entry. There is no good reason to install 3.19.0 if you have 3.19.3 or .4

**Comment by tjhei** [comment]
Can you also add the 3.18.x versions, please? I think we need to switch to 3.18 by default to fix #358 .

File: deal.II-toolchain/packages/petsc.package (hunk 1 above)
Comment (alarshi) [reply deal.II-toolchain/packages/petsc.package]: Absolutely!

File: deal.II-toolchain/packages/petsc.package (hunk 2 above)
Comment (alarshi) [reply deal.II-toolchain/packages/petsc.package]: Got it.

**Comment by alarshi** [comment]
Hi Timo, thank you for the suggestions! I have addressed your comments in the latest commit.

**Comment by tjhei** [comment]
Thanks. Slepc also needs to be downgraded to the latest 3.18.x for them to work.

**Comment by alarshi** [comment]
Got it, just pushed the changes.

**Review by tjhei — APPROVED** [review]

**Pull request merged**